include/rocksdb/iostats_context.h

```diff
@@ -50,7 +50,15 @@ struct IOStatsContext {
   uint64_t cpu_read_nanos;
 };
 
-// Get Thread-local IOStatsContext object pointer
+// If RocksDB is compiled with -DNIOSTATS_CONTEXT, then a pointer to a global,
+// non-thread-local IOStatsContext object will be returned. Attempts to update
+// this object will be ignored, and reading from it will also be no-op.
```

Comment:
Is the "ignored and reading" true or is it "undefined"?

Reply:
The method will be no-op, with deterministic execution result.